src/shared/scrapers/ES/index.js

```diff
@@ -1,12 +1,35 @@
+import assert from 'assert';
 import datetime from '../../lib/datetime/iso/index.js';
 import * as fetch from '../../lib/fetch/index.js';
 import * as transform from '../../lib/transform.js';
 import * as parse from '../../lib/parse.js';
 
 const { looksLike } = datetime;
 
+const countryLevelMap = {
+  Andalucía: 'iso2:ES-AN',
+  Aragón: 'iso2:ES-AR',
```

Comment:
is it worth deburring these before matching?

Reply:
What is deburring?

Reply:
tl;dr nearest ascii conversion